mips/switch.S

```diff
@@ -39,7 +39,7 @@ ctx_resume:
         LOAD_PCPU($t0)
         sw      $s0, PCPU_CURTHREAD($t0)
 
-        lw      $a0, TD_USPACE($s0)
+        lw      $a0, PCPU_USPACE($t0)
```

Comment:
I'm not sure that works at all? Instead of reading which `vm_map` this thread uses, you just activate the one that is currently active. That doesn't change the current map at all, right? Or do I miss something?